### PR TITLE
Gh 398 different input types

### PR DESCRIPTION
**Pull request by d47853**

### CODE REVIEW

File: ui/src/test/webapp/app/query/query-component-spec.js [hunk 1]
```diff
@@ -664,26 +664,138 @@ describe('The Query component', function() {
                 });
             });
 
-            describe('When adding seeds', function() {
+            describe('When adding seed pairs', function() {
```
Comment (p013570) [inline ui/src/test/webapp/app/query/query-component-spec.js]: Can we keep the normal seed tests in too?

File: ui/src/main/webapp/app/gaffer/operation-service.js [hunk 2]
```diff
@@ -45,12 +45,23 @@ angular.module('app').factory('operationService', ['$http', '$q', 'settings', 'c
         return deferredAvailableOperations.promise;
     }
 
+    var hasInputB = function(first) {
+        return common.endsWith(first, "GetElementsBetweenSets");
+    }
+
+    var getInputType = function(first) {
+        if (common.endsWith(first, 'GetElementsInRanges')) {
+            return 'uk.gov.gchq.gaffer.commonutil.pair.Pair';
```
Comment (p013570) [inline ui/src/main/webapp/app/gaffer/operation-service.js]: This shouldn't be hardcoded. We should look this up in the json config.

File: ui/src/main/webapp/app/gaffer/operation-service.js [hunk 3]
```diff
@@ -45,12 +45,23 @@ angular.module('app').factory('operationService', ['$http', '$q', 'settings', 'c
         return deferredAvailableOperations.promise;
     }
 
+    var hasInputB = function(first) {
+        return common.endsWith(first, "GetElementsBetweenSets");
```
Comment (p013570) [inline ui/src/main/webapp/app/gaffer/operation-service.js]: This also shouldn't be hardcoded, it should be fetched from the json config.

File: ui/src/test/webapp/app/query/query-component-spec.js (hunk 1 above)
Comment (d47853) [reply ui/src/test/webapp/app/query/query-component-spec.js]: normal tests are just below in the file

**Pull request merged**